tools/tpm2_sign.c

```diff
@@ -262,6 +276,15 @@ static bool init(int argc, char *argv[], tpm_sign_ctx *ctx) {
             flags.s = 1;
         }
             break;
+        case 'r': {
+            bool result = files_does_file_exist(optarg);
```

Comment:
I also dislike how the tools complain that a file exists, it should just overwrite it.